subprojects/core/src/integTest/groovy/org/gradle/api/internal/project/ant/AntImportBuildIntegrationTest.groovy

```diff
@@ -0,0 +1,108 @@
+/*
+ * Copyright 2020 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradle.api.internal.project.ant
+
+import groovy.xml.MarkupBuilder
+import org.gradle.integtests.fixtures.AbstractIntegrationSpec
+import spock.lang.Issue
+
+class AntImportBuildIntegrationTest extends AbstractIntegrationSpec {
+
+    private static final String EXECUTER_SYS_PROP = "org.gradle.integtest.executer"
+    private static final String EXECUTER_CONFIG_CACHE = "configCache"
+    private static final String EXECUTER_FORKING = "forking"
+
+    private static String initialExecuterType
+
+    File antBuildFile
+
+    /**
+     * Ensure that {@link org.gradle.integtests.fixtures.executer.ConfigurationCacheGradleExecuter}
+     * will not be created, because of serialization issues with {@link org.gradle.api.tasks.ant.AntTarget}.
+     */
+    def setupSpec() {
+        initialExecuterType = System.getProperty(EXECUTER_SYS_PROP)
+        if (initialExecuterType == EXECUTER_CONFIG_CACHE) {
+            System.setProperty(EXECUTER_SYS_PROP, EXECUTER_FORKING)
+        }
+    }
+
+    /**
+     * Restore initial property value.
+     */
+    def cleanupSpec() {
+        if (initialExecuterType != null) {
+            System.setProperty(EXECUTER_SYS_PROP, initialExecuterType)
+        }
+    }
```

Comment:
@rieske There's a better way to do this right?

Reply:
Tests that fail with configuration cache should be annotated with `@ToBeFixedForConfigurationCache(because="<reason>")` instead.